### ISSUE TITLE
Shortcuts translation

### ISSUE THREAD

**Issue by hugok79** [elementary/switchboard-plug-keyboard#368]
Where in "Weblate" is this translation. What is the resource?

![Captura de ecrã de 2021-06-17 10-58-13](https://user-images.githubusercontent.com/16929048/122384698-79f4ec00-cf5b-11eb-9ac1-a0890fce862e.png)

**Comment by hugok79** [comment]
In English
![Screenshot from 2021-06-17 12-00-07](https://user-images.githubusercontent.com/16929048/122392975-ea077000-cf63-11eb-9543-8c50221b2edb.png)

**Comment by hugok79** [comment]
Where is the  "⌘ + End" source translation?

**Comment by hugok79** [comment]
And where is the Crtl + Alt + Delete source translation?
![Screenshot from 2021-06-17 12-00-27](https://user-images.githubusercontent.com/16929048/122393647-9ea19180-cf64-11eb-934c-fa348c870edd.png)

**Comment by ryonakano** [comment]
Maybe in granite?

**Comment by ryonakano** [comment]
https://l10n.elementary.io/translate/desktop/granite/pt/?checksum=30924bfbd4e5a4b7&sort_by=-priority,position

**Comment by hugok79** [comment]
Thanks for reply @ryonakano
Bu there is no "Delete"! Only "Crtl" and "Alt" appears..... The same goes for "End"

**Comment by ryonakano** [comment]
Hmm, "Delete" does not seem to be marked as translatable in granite. Do "Delete" and "End" need translated in Portuguese?

**Comment by tintou** [comment]
The "Delete" word is translated at GTK side, here is for Portuguese: https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/po/pt.po#L292

**Comment by hugok79** [comment]
> Hmm, "Delete" does not seem to be marked as translatable in granite. Do "Delete" and "End" need translated in Portuguese?

No, even because the key labels on the Portuguese keyboard are in English...
The problem is that this appears translated into Portuguese and that is not the intended...
Picture of my Sony Vaio keys:
![IMG_0104](https://user-images.githubusercontent.com/16929048/122399672-e7f8dd80-cf72-11eb-8bc3-a7cfb3c2b2a0.JPG)

**Comment by hugok79** [comment]
> The "Delete" word is translated at GTK side, here is for Portuguese: https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/po/pt.po#L292

Thanks @tintou I am also the GNOME coordinator and translator... But also there the Portuguese translation is "Delete"
That was translated by me.... I am also the GNOME coordinator and translator 👍
